<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Painting Board</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="Canvas" name="canvas">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>924</width>
         <height>601</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionHorizontal_Flip"/>
    <addaction name="actionVertical_Flip"/>
    <addaction name="separator"/>
    <addaction name="actionInvert_Color"/>
    <addaction name="actionGrayscale"/>
    <addaction name="actionEqualize_Hist"/>
    <addaction name="separator"/>
    <addaction name="actionBlur"/>
    <addaction name="actionMedian_Blur"/>
    <addaction name="actionGaussian_Blur"/>
    <addaction name="separator"/>
    <addaction name="actionSepia"/>
    <addaction name="actionSharpen"/>
    <addaction name="actionPixelize"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced</string>
    </property>
    <addaction name="actionDerain"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAdvanced"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actiondefault"/>
   <addaction name="actionPen"/>
   <addaction name="actionspray"/>
   <addaction name="actionErase"/>
   <addaction name="acitonFill"/>
   <addaction name="actionLine"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="drawingToolbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="viewToolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFitWindow"/>
   <addaction name="actionFitWidth"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/svg/resources/open.svg</normaloff>:/svg/resources/open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/svg/resources/save.svg</normaloff>:/svg/resources/save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInvert_Color">
   <property name="text">
    <string>&amp;Invert Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionHorizontal_Flip">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/svg/resources/flip_horizontal.svg</normaloff>:/svg/resources/flip_horizontal.svg</iconset>
   </property>
   <property name="text">
    <string>Horizontal Flip</string>
   </property>
  </action>
  <action name="actionVertical_Flip">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/svg/resources/flip_vertical.svg</normaloff>:/svg/resources/flip_vertical.svg</iconset>
   </property>
   <property name="text">
    <string>Vertical Flip</string>
   </property>
  </action>
  <action name="actionFitWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/svg/resources/fit_window.svg</normaloff>:/svg/resources/fit_window.svg</iconset>
   </property>
   <property name="text">
    <string>fit window</string>
   </property>
  </action>
  <action name="actionFitWidth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/svg/resources/fit_width.svg</normaloff>:/svg/resources/fit_width.svg</iconset>
   </property>
   <property name="text">
    <string>fit width</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/svg/resources/new.svg</normaloff>:/svg/resources/new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEqualize_Hist">
   <property name="text">
    <string>Equalize Hist</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="actionMedian_Blur">
   <property name="text">
    <string>Median Blur</string>
   </property>
  </action>
  <action name="actionSharpen">
   <property name="text">
    <string>Sharpen</string>
   </property>
  </action>
  <action name="actionGaussian_Blur">
   <property name="text">
    <string>Gaussian Blur</string>
   </property>
  </action>
  <action name="actionSepia">
   <property name="text">
    <string>Sepia</string>
   </property>
  </action>
  <action name="actionPixelize">
   <property name="text">
    <string>Pixelize</string>
   </property>
  </action>
  <action name="actionDerain">
   <property name="text">
    <string>Derain</string>
   </property>
  </action>
  <action name="actionShadowRemoval">
   <property name="text">
    <string>Shadow Removal</string>
   </property>
  </action>
  <actiongroup name="GuiActions">
   <action name="actionLine">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/svg/resources/line.svg</normaloff>:/svg/resources/line.svg</iconset>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </action>
   <action name="actionRectangle">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/svg/resources/rectangle.svg</normaloff>:/svg/resources/rectangle.svg</iconset>
    </property>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </action>
   <action name="actionEllipse">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/svg/resources/circle.svg</normaloff>:/svg/resources/circle.svg</iconset>
    </property>
    <property name="text">
     <string>Ellipse</string>
    </property>
   </action>
   <action name="actiondefault">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/svg/resources/Cursor.svg</normaloff>:/svg/resources/Cursor.svg</iconset>
    </property>
    <property name="text">
     <string>default</string>
    </property>
   </action>
   <action name="actionErase">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/svg/resources/eraser.svg</normaloff>:/svg/resources/eraser.svg</iconset>
    </property>
    <property name="text">
     <string>Erase</string>
    </property>
   </action>
   <action name="acitonFill">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/svg/resources/fill.svg</normaloff>:/svg/resources/fill.svg</iconset>
    </property>
    <property name="text">
     <string>Fill</string>
    </property>
   </action>
   <action name="actionPen">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/svg/resources/brush.svg</normaloff>:/svg/resources/brush.svg</iconset>
    </property>
    <property name="text">
     <string>Pen</string>
    </property>
   </action>
   <action name="actionspray">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/svg/resources/spray.svg</normaloff>:/svg/resources/spray.svg</iconset>
    </property>
    <property name="text">
     <string>spray</string>
    </property>
   </action>
  </actiongroup>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
